keps/0009-operator-toolkit.md

```diff
@@ -106,93 +100,80 @@ An operator package is a folder that contains all of the manifests needed to cre
 
 #### operator.yaml
 
-`operator.yaml` is the base definition of an operator. It follows the following format, extracted from the MySQL example:
+`operator.yaml` is the base definition of an operator.  It follows the following reference format.
 
 ```yaml
-name: operator
-description: operator
-version: "5.7"
-kudoVersion: ">= 0.2.0"
-kubeVersion: ">= 1.14"
+name: str(max=253, lowercase alphanumeric characters, -, and .)
+description: str(required=False)
+version: semver(desc='version of operator')
+appVersion: str(desc='component controlled version', required=False)
+kudoVersion: semver(desc='minVersion of KUDO')
+kubernetesVersion: semver(desc='minVersion of Kubernetes')
```

Comment:
what about - 'minimal required version (semver) of Kubernetes required for this operator to work'